lib/executor.js

```diff
@@ -116,7 +116,9 @@ function setPort(app, instructions) {
     process.env.npm_package_config_port,
     app.get('port'),
     3000
-  ], _.isFinite);
+  ], function(v) {
+    return _.isFinite(parseInt(v, 10));
+  });
```

Comment:
Why is this change necessary? If I am reading the docs correctly, then lodash isFinite supports string values too. Note that `parseInt('10asd', 10)` return `10`, which is probably not what we want.